app/gui2/src/components/GraphEditor/widgets/WidgetArgumentName.vue

```diff
@@ -34,7 +34,7 @@ export const widgetDefinition = defineWidget([ArgumentPlaceholder, ArgumentAst],
 <template>
   <span class="WidgetArgumentName" :class="{ placeholder, primary }">
     <template v-if="showArgumentValue">
-      <span class="name">{{ props.input.info!.name }}</span
+      <span class="value">{{ props.input.info!.name }}</span
```

Comment:
Not sure why is this changed? That span is actually representing the argument name, not its value. The "value" is represented by the `NodeWidget` below.